Worksheet_B/README.MD

```diff
@@ -0,0 +1,12 @@
+```
+Player Starts Minigame
+A five Letter word is randomly generated
+Player has 4 tries to guess the correct code
+game ends after 4 incorrect tries or the correct guess
+if the player guesses correctly then open door or chest
+if generated word is "grass" and player guesses "glass" remove a try and show a likeness of 4
+The likeness is how many letters are in both the players guess and the generated word
+Only reset the word after the game has been
+```
+
+![Flowchart](flowchart.pdf)
```

Comment:
Flowchart needs arrows on the lines to show the direction of flow, but otherwise looks OK